addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -155,8 +196,8 @@ QUnit.test('composer text input: basic rendering when linked thread is a mail.ch
     );
     assert.strictEqual(
         document.querySelector(`.o_ComposerTextInput_textarea`).placeholder,
-        "Write something...",
-        "should have 'Write something...' as placeholder in composer text input if composer is for a 'mail.channel'"
```

Comment:
Remove this part from the "basic rendering" test, and make new specific tests with just 1 assertion each to verify the placeholder depending on channel type.